crates/node/src/evm.rs

```diff
@@ -11,30 +11,35 @@
 //! precompiles defined by [`alphanet_precompile`].
 
 use alphanet_precompile::secp256r1;
-use reth::{
-    primitives::{
-        revm_primitives::{CfgEnvWithHandlerCfg, TxEnv},
-        transaction::FillTxEnv,
-        Address, Bytes, Header, TransactionSigned, TxKind, U256,
-    },
-    revm::{
-        handler::register::EvmHandler,
-        inspector_handle_register,
-        precompile::PrecompileSpecId,
-        primitives::{AnalysisKind, Env, OptimismFields},
-        ContextPrecompiles, Database, Evm, EvmBuilder, GetInspector,
-    },
-};
 use reth_chainspec::{ChainSpec, EthereumHardfork, Head, OptimismHardfork};
 use reth_node_api::{ConfigureEvm, ConfigureEvmEnv};
+use reth_primitives::{
+    revm_primitives::{CfgEnvWithHandlerCfg, TxEnv},
+    transaction::FillTxEnv,
+    Address, Bytes, Header, TransactionSigned, TxKind, U256,
+};
+use reth_revm::{
+    handler::register::EvmHandler,
+    inspector_handle_register,
+    precompile::PrecompileSpecId,
+    primitives::{AnalysisKind, Env, OptimismFields},
+    ContextPrecompiles, Database, Evm, EvmBuilder, GetInspector,
+};
 use std::sync::Arc;
 
 /// Custom EVM configuration
-#[derive(Debug, Clone, Copy, Default)]
+#[derive(Debug, Clone, Default)]
 #[non_exhaustive]
```

Comment:
```suggestion
```

Reply:
just a note here that this is also present in upstream: https://github.com/paradigmxyz/reth/blob/acdb7b7e2ea2c3d0ba8c4a330fd1e9a5468a85fa/crates/optimism/evm/src/lib.rs#L36

Reply:
https://github.com/paradigmxyz/reth/pull/10865

Reply:
resolved